tests/unit/TodoList.spec.ts

```diff
@@ -1516,6 +1516,61 @@ describe('TodoList - Click Tests', () => {
       done()
     })
   })
+
+  it('Confirm BCOL error is captured in todo list', async () => {
+    sessionStorage.setItem('PAY_API_URL', '') 
+    store.state.tasks = [
+      {
+        'task': {
+          'filing': {
+            'header': {
+              'name': 'annualReport',
+              'ARFilingYear': 2019,
+              'status': 'DRAFT',
+              'filingId': 123,
+              'paymentErrorType': 'BCOL_ERROR'
+            },
+            'annualReport': {
+              'annualGeneralMeetingDate': '2019-07-15',
+              'annualReportDate': '2019-07-15'
+            },
+            'changeOfAddress': {},
+            'changeOfDirectors': {}
+          }
+        },
+        'enabled': true,
+        'order': 1
+      }
+    ]
+
+    sinon.stub(axios, 'get').withArgs('codes/errors/BCOL_ERROR')
+    .returns(new Promise(resolve => resolve({ data: {
+      detail: 'An Error has occured',
+      title: 'Error'
+    }})))
+
+    const wrapper = mount(TodoList, { store, vuetify })
+    const vm = wrapper.vm as any
+
+    expect(vm.$el.querySelector('.bcol-error')).toBeNull()
+
+    await flushPromises()
+
+    await Vue.nextTick()
+
+    const todoItem = vm.$el.querySelector('.bcol-error')
+
+    const button = wrapper.find('.expand-btn')
+
+    button.trigger('click')
```

Comment:
Great unit test! Could you please add a few comments to narrate what the test is doing (as with other unit tests)? Thx.

Reply:
done

Reply:
Fixed.